docs/janssen-server/developer/external-libraries.md

```
@@ -8,12 +8,9 @@ tags:
 
 # Using External Libraries In Interception Scripts
 
-Janssen Server [Interception script](./interception-scripts.md) is a very powerful tool that enables developers to 
+Janssen Server [Interception script](../../janssen-server/developer/scripts/README.md) is a very powerful tool that enables developers to 
 control many different aspects of the authentication process.
 
 It is possible to use external libraries in interception scripts. It is possible to use external libraries Java 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)